synapse/rest/client/room.py

```diff
@@ -60,6 +62,30 @@
 
 logger = logging.getLogger(__name__)
 
+messsages_response_timer = Histogram(
+    "synapse_room_message_list_rest_servlet_response_time_seconds",
+    "sec",
+    [],
+    buckets=(
+        0.005,
+        0.01,
+        0.025,
+        0.05,
+        0.1,
+        0.25,
+        0.5,
+        1.0,
+        2.5,
+        5.0,
+        10.0,
+        30.0,
+        60.0,
+        120.0,
+        180.0,
```

Comment:
Note: with such large buckets the resolution is going to be pretty low for anything above 10s. This is probably fine, but does mean that to move the needle meaningfully on this graph `/messages` timings would need to change by tens of seconds.

Reply:
I think we're good for now. Didn't want to be too greedy with the number of buckets.

Reply:
Cool, like i said I'm way more blase of increasing the buckets on one of histograms

Reply:
Increased fidelity in https://github.com/matrix-org/synapse/pull/13584#discussion_r951756543